src-built-in/components/userPreferences/src/components/content/Workspaces.jsx

```diff
@@ -108,6 +110,24 @@ export default class Workspaces extends React.Component {
 		UserPreferencesStore.addListener({ field: 'WorkspaceList' }, this.setWorkspaceList)
 	}
 
+	/**
+	 * Sets the windowOptions.alwaysOnTop to the value supplied. The preferences component should
+	 * not be always on top when a file dialog is open.
+	 * @param {boolean} alwaysOnTop The value to set finsembleWindow.options.alwaysOnTop
+	 */
+	changeAlwaysOnTop(alwaysOnTop) {
```

Comment:
If this is only used for preferences or even for dialogs, I'd suggest renaming this. It's a little confusing why this is called in the code with a generic name.